tags/not_support.md

```diff
@@ -0,0 +1,13 @@
+---
+title: 🚨 This is not a support channel! 🚨 
+color: orange
+---
+
```

Comment:
```suggestion
This channel is not designated as a support channel!

To find support for your problem(s), go to the relevant channel from below:

```

Reply:
I agree, this isn't needed with the title.